conda-forge.yml

```diff
@@ -12,3 +12,7 @@ provider:
   linux_aarch64: azure
   linux_ppc64le: azure
 test: native_and_emulated
+conda_install_tool: pixi
+conda_build_tool: rattler-build
+bot:
+  automerge: true
```

Comment:
Please remove the bot automerge feature. This prevents us from having time to review the recipe and potentially updating the dependencies.